marklogic-data-hub-central/ui/src/components/modeling/entity-type-table/entity-type-table.test.tsx

```diff
@@ -28,11 +28,11 @@ describe('EntityTypeModal Component', () => {
 
       expect(getByText(/Customer/i)).toBeInTheDocument();
       expect(getByText(/1,000/i)).toBeInTheDocument();
-      expect(getByText(/2020-04-09 14:27/i)).toBeInTheDocument();
+      expect(getByText(/09-Apr-20 14:27/i)).toBeInTheDocument();
```

Comment:
@wooldridge  @brucean52 Is DD-MMM-YY the standard for date formatting in the UI? I'm a big fan of YYYY-MM-DD because it's naturally sortable, but many users I've worked with prefer something else, which always disappoints me. Do we have an agreed-upon standard yet for HC?

Reply:
Let me get confirmation on this. The modeling design wiki shows YYYY-MMM-DD. The figma design shows the changes I made here

Reply:
Last processed now shows time relative time in the table, which is sortable. Hover text over time shows date in DD-MMM-YY